Source/SIMPLib/CoreFilters/CreateDataArray.cpp

```diff
@@ -56,89 +56,97 @@
+  struct From_##type                                                                                                                                                                                   \
+  {                                                                                                                                                                                                    \
+    type convert(const QString& input)                                                                                                                                                                 \
+    {                                                                                                                                                                                                  \
+      bool ok = false;                                                                                                                                                                                 \
+      type value = static_cast<type>(input.func(&ok));                                                                                                                                                 \
+      if(!ok)                                                                                                                                                                                          \
+      {                                                                                                                                                                                                \
+        return 0;                                                                                                                                                                                      \
+      }                                                                                                                                                                                                \
+      return value;                                                                                                                                                                                    \
+    }                                                                                                                                                                                                  \
+  };
+
+CDA_CREATE_CONVERTOR(uint8_t, toUInt)
+CDA_CREATE_CONVERTOR(int8_t, toInt)
+CDA_CREATE_CONVERTOR(uint16_t, toUShort)
+CDA_CREATE_CONVERTOR(int16_t, toShort)
+CDA_CREATE_CONVERTOR(uint32_t, toUInt)
+CDA_CREATE_CONVERTOR(int32_t, toInt)
+CDA_CREATE_CONVERTOR(uint64_t, toULongLong)
+CDA_CREATE_CONVERTOR(int64_t, toLongLong)
+CDA_CREATE_CONVERTOR(bool, toInt)
+
+CDA_CREATE_CONVERTOR(float, toDouble)
+CDA_CREATE_CONVERTOR(double, toDouble)
+
+#define CDA_SEP ";"
+#define CDA_LB "["
+#define CDA_RB "]"
```

Comment:
```suggestion
constexpr char CDA_SEP = ';';
constexpr char CDA_LB = '[';
constexpr char CDA_RB = ']';
```
Change to constexpr char variables.